README.md

```diff
@@ -57,13 +57,18 @@ $ make build
 
 Connect the esp32 board to your computer using the USBC connection port on the esp32 and a cable that will connect to your computer.
 
-Press and hold the "boot" button then press the "reset" button and release both at the same time (shown below). This puts the esp32 into a flashing mode.
+Press and hold the "boot" button, then press and release the "reset" button, then release the "boot" button (shown below). This puts the esp32 into a flashing mode. If done correctly, the lights should turn off.
```

Comment:
Ah ok. Is there any other reliable indicator of when it's in DFU mode?

Reply:
Hey @keerats7 ! As it mentions above, you will see the sensor power lights turn off. The device will also enumerate differently to the system. Something like ttymodem etc instead of frogsensor.

Reply:
Hey @keenanjohnson , based on @damz comment it seems the CHG LED on the ESP32 may turn off, but that's unrelated to it being in DFU mode. Are you saying the green sensor lights will always turn off when in DFU mode? If so I can add that distinction that specifically the sensor lights will turn off.